walnuts_cpp/examples/test_stan.cpp

```diff
@@ -1,26 +1,27 @@
-#include <walnuts/nuts.hpp>
-#include <Eigen/Dense>
 #include <bridgestan.h>
+
+#include <Eigen/Dense>
+#include <chrono>
+#include <cmath>
 #include <iostream>
 #include <memory>
 #include <random>
-#include <cmath>
-#include <chrono>
-
+#include <walnuts/nuts.hpp>
+#include <walnuts/walnuts.hpp>
 
 // consider using something like https://github.com/martin-olivier/dylib/
 #ifdef _WIN32
```

Comment:
Can you put all of this into its own .hpp file? Or is this just a prototype?

Reply:
This is still pretty prototype-y. In the real code I’d prefer to use a library to handle the platform differences for dlopen

Reply:
All good